docusaurus/docs/snippets/managing-relations.md

```diff
@@ -0,0 +1 @@
+In the `data` object, relations can be managed with the `connect`, `disconnect`, and `set` parameters using the syntax described for the REST API (see [managing relations](/dev-docs/api/rest/relations)).
```

Comment:
This link seems broken, but maybe it is related to the upcoming relations docs release? I think there is another one below as well.

Reply:
It indeed will work only when the new "Managing relations" page (part of the relations reordering release) is also converted to Docusaurus. @gpene, I'd say it's best to comment out the various places where the snippet is inserted, to keep things clean and be able to ship this PR as soon as approved.
We'll create another PR to convert the new page and uncomment these lines.